cypress/integration/admin/login/login.spec.js

```diff
@@ -17,4 +19,146 @@ context("Login Page", () => {
             cy.findByText(Cypress.env("DEFAULT_ADMIN_USER_USERNAME")).should("exist");
         });
     });
+
+    it("try to access admin pages as anonymous user", () => {
+        cy.visit("/page-builder/pages");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.visit("/form-builder/forms");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.visit("/cms/content-models");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+    });
+
+    it("try to access non-existing page as logged user", () => {
+        cy.visit("/");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.findByLabelText(/your e-mail/i).type("admin@webiny.com");
+        cy.findByLabelText(/your password/i).type("12345678");
+        cy.findByText(/submit/i).click();
+
+        cy.findByPlaceholderText(/search.../i).should("exist");
+        cy.findByText(/pages/i).should("exist");
```

Comment:
Can we replace this portion of the code with just the `cy.login` call that we're using in other tests?

Reply:
The same question goes for `authenticated user must logout successfully` test.

Reply:
Amazing idea! I'm on it.